pkg/ami/static_resolver_ami.go

```diff
@@ -1,21 +1,18 @@
+package ami
```

Comment:
I think we should make sure in this generated file that we keep the comment header that indicates its a generated file as per convention:

`// Code generated by go generate; DO NOT EDIT.`

Reply:
Yes, good idea, thanks!